2_deploy_ses_aio/roles/setup_ses_configs/templates/ses-config.yml.j2

```diff
@@ -0,0 +1,28 @@
+ses_cluster_configuration:
```

Comment:
I would let this var be part of the playbooks instead as it seems more nicer there to point out what is loading as reference rather than hidden in this file which would make it difficutl to know where the data is coming from.

Reply:
I guess it depends how the "official" ses_config looks like

Reply:
Not sure if there is anything really "official". The stuff coming out of the salt "openstack.integrate" function looks somewhat different (ideally we would be using that).
This PR currently is basically using the same output  as: https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/ardana/ansible/roles/ses_ardana_integration/tasks/main.yml

Reply:
Not sure if I am a using the right prefix here, but I guess you get the idea:

```
rhafer-ses:~ # salt-run --out=yaml openstack.integrate prefix=cloud
ceph_conf:
  cluster_network: 192.168.100.0/24
  fsid: 949e7d17-16da-37a9-b84a-ce6b52faee1e
  mon_host: 192.168.100.7
  mon_initial_members: rhafer-ses
  public_network: 192.168.100.0/24
cinder:
  key: AQBWm1pcAAAAABAAqyoU6usgH2YzmASRFZOdCg==
  rbd_store_pool: !!python/unicode cloud-cloud-volumes
  rbd_store_user: !!python/unicode cloud-cinder
cinder-backup:
  key: AQBWm1pcAAAAABAAWUYFZbTyLc4a3ElHSiuY+w==
  rbd_store_pool: !!python/unicode cloud-cloud-backups
  rbd_store_user: !!python/unicode cloud-cinder-backup
glance:
  key: AQBWm1pcAAAAABAAJOCuucpxBacq5JvfB1XKkA==
  rbd_store_pool: !!python/unicode cloud-cloud-images
  rbd_store_user: !!python/unicode cloud-glance
nova:
  rbd_store_pool: !!python/unicode cloud-cloud-vms
radosgw_urls:
- http://rhafer-ses:8080/swift/v1
```

Reply:
The ses role itself currently doesn't generate any ses_config yml. It just registers the `conf_options` and  `ceph_files` variables.

There is a another variant for import ses_config (based on "openstack.integrate") in the works for ardana currently, but that is currently blocked by a bug in deepseaa it seem. See: https://github.com/SUSE-Cloud/automation/pull/3121

Reply:
So what we can do is indeed (just an idea of the code):
```
- name: Run openstack.integrate
  command: salt-run --out=yaml openstack.integrate prefix=cloud
  register: _saltrunopenstackintegrate
  when:  ### Ensure here it runs only on the first node of the cluster.
### Ensure the yaml is loaded in memory on that node
- name: Manipulate ses_config in userspace 
  set_fact:
    ses_config_file: "{{ _saltrunopenstackintegrate.stdout | from_yaml }}" ## Also add a top level item, and reoutput to nice yaml. Alternatively manipulate the string as json for making it easier.

- name: Drop ses_config in temp usersave
  copy:
    content: "{{ ses_config_file }}"
    dest: ### what we've decided as temp location, but in the socok8s folder, so that there is automatic deletion, and we don't have this kind of errors anymore.
  delegate_to: localhost
```

Reply:
or just borrow from the ardana integration: https://github.com/SUSE-Cloud/automation/blob/5a111ad735b324752190005f285d6e1275469f4f/scripts/jenkins/ardana/ansible/roles/ses_ardana_integration/tasks/ses_configure_v2.yml

As said the main issue seems to be that "openstack.integrate" is currently broken.

Reply:
Oh sorry, I missed that openstack.integrate is broken.